apps/req.c

```diff
@@ -127,7 +127,7 @@ const OPTIONS req_options[] = {
     {"subj", OPT_SUBJ, 's', "Set or modify request subject"},
     {"subject", OPT_SUBJECT, '-', "Output the request's subject"},
     {"multivalue-rdn", OPT_MULTIVALUE_RDN, '-',
-     "Enable support for multivalued RDNs"},
+     "This has become obsolete since support for multi-valued RDNs is now always on."},
```

Comment:
I think it is a little bit too verbose. What about `Unused, multi-valued RDNs support is always on`

Reply:
Just say "(deprecated)"  That's what is done for ```OPT_V_OPTIONS``` which is used by eight commands. Be consistent.  Put more text in the docs if needed.

Reply:
So I've changed both occurrences of the text to
"Deprecated; multi-valued RDNs support is always on."
using "deprecated" rather than "obsolete" for consistency,
and keeping "support for multi-valued RDNs is now always on"
to make clear that not the feature itself is deprecated (but just its activation).

Reply:
I do not see this change being pushed.

Reply:
Sorry, I forgot to do that. Done now.